src/routes/auth.js

```diff
@@ -0,0 +1,12 @@
+const M = require('../middle-end');
+const { default: connectedAuthWrapper } = require('redux-auth-wrapper/connectedAuthWrapper');
+const NotFoundPage = require('../components/NotFoundPage');
+
+exports.authenticate = connectedAuthWrapper({
+    wrapperDisplayName: 'UserIsAuthenticated',
+    authenticatedSelector: (state) => {
```

Comment:
I expect this will flicker the 404 route while the initial login action completes.  Should be able to address this by configuring `authenticatingSelector`.

Reply:
That's totally right. In other projects, we leverage `strange-auth`'s statuses for that selector, but I'll see what we can do here.